homeassistant/components/watergate/sensor.py

```diff
@@ -0,0 +1,215 @@
+"""Support for Watergate sensors."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from enum import StrEnum
+import logging
+
+from homeassistant.components.sensor import (
+    HomeAssistant,
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+    UnitOfPressure,
+    UnitOfTemperature,
+    UnitOfTime,
+    UnitOfVolume,
+    UnitOfVolumeFlowRate,
+)
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import dt as dt_util
+
+from . import WatergateConfigEntry
+from .coordinator import WatergateAgregatedRequests, WatergateDataCoordinator
+from .entity import WatergateEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 0
+
+
+class PowerSupplyMode(StrEnum):
+    """LED bar mode."""
+
+    BATTERY = "battery"
+    EXTERNAL = "external"
+    BATTERY_EXTERNAL = "battery+external"
```

Comment:
```suggestion
    BATTERY_EXTERNAL = "battery_external"
```
I'm not sure if `strings` would work nicely with the "+"

Reply:
I would indeed use the underscore